bundle/run.sh

```diff
@@ -56,6 +57,8 @@ export SECRETS_DIRECTORY=/data/secrets
 export OAUTH_ID_TOKEN_KEY_PATH=${OAUTH_KEYS_PATH}/id-token.pem
 export OAUTH_ACCESS_TOKEN_KEY_PATH=${OAUTH_KEYS_PATH}/access-token.pem
 
+export OAUTH_DEVICE_SECRET_PATH=${OAUTH_SECRETS_PATH}/device.secret
```

Comment:
please update this doc https://plgd.dev/tutorials/external-oauth-server/

Reply:
I think @Danielius1922 will update it because he removes lots of ENV vars.